### DISCUSSION TITLE
[GENERAL] Why ranged weapons are not all in the same InventoryType?

### DISCUSSION THREAD

**Discussion by BarbzYHOOL** [azerothcore/azerothcore-wotlk#3903] [General]
In item_template, there are 2 "ranged" slots (in InventoryType column).

In the first one there are only BOWS.
In the second one, there are guns, crossbows and wands.

Wtf is that?

Is it because we have to follow this for the client to work correctly and thus we can only blame blizzard's developers? or can we improve this? 

EDIT: i just tried to change a bow inventory type from 15 to 26 (gun/wand) and i can equip it  but i have not learned "gun", so i'm really wondering if we couldn't unify the values for less confusion, less wasted time, and easier "ORDER BY" queries

**Comment by Helias** [comment]
probably for DBCs? did you check itemClass, itemSubclass?

**Comment by BarbzYHOOL** [comment]
what? i don't think it's related
